mosaic_creator.py

```diff
@@ -11,17 +12,39 @@
 def getImages(images_directory):
     files = os.listdir(images_directory)
     images = []
-    for file in files:
-        filePath = os.path.abspath(os.path.join(images_directory, file))
-        try:
-            fp = open(filePath, "rb")
-            im = Image.open(fp)
-            images.append(im)
-            im.load()
-            fp.close()
-        except:
-            pass
-    return (images)
+    if files:
+        for file in files:
+            filePath = os.path.abspath(os.path.join(images_directory, file))
+            try:
+                fp = open(filePath, "rb")
+                im = Image.open(fp)
+                images.append(im)
+                im.load()
+                fp.close()
+            except FileNotFoundError:
+                print("Can not find the specified image file. Please, make sure the passed file exists.")
+                sys.exit()
+            except ValueError:
+                print("Invalid parameters passed to PIL.Image.open() method.")
+                sys.exit()
+            except UnidentifiedImageError:
+                #If cant open the file, ignore it and continue
+                print("WARNING: Can not identify and open", file, "Ignoring this file...")
+                continue
+            except IsADirectoryError:
+                print("Argument passed to --images is a directory without any files. Image files path expected.")
+                sys.exit()
+        
+        #When there is no image files in the directory, but it has some other files (pdf, txt, etc), 
+        #files list will be empty
+        if images:
```

Comment:
This is not required. We can simply use glob to check the number of files with *png or *jpg in the given path